internal/commands/uninstall.go

```diff
@@ -41,6 +47,17 @@ func runUninstall(dockerCli command.Cli, installationName string, opts credentia
 	if err != nil {
 		return err
 	}
+	if opts.force {
+		defer func() {
+			if mainErr == nil {
+				return
+			}
+			if err := installationStore.Delete(installationName); err != nil {
+				fmt.Fprintf(os.Stderr, "failed to force deletion of installation %q: %s\n", installationName, err)
+			}
+			fmt.Fprintf(os.Stderr, "deletion forced for installation %q\n", installationName)
```

Comment:
I think the ordering of these two messages will be backwards in the case where they are both printed:
```
failed to force deletion of installation "foo": something went wrong
deletion forced for installation "foo"
```
I think you should move this second print up so the output would be:
```
deletion forced for installation "foo"
failed to force deletion of installation "foo": something went wrong
```

Should we also log `mainErr` or does that already happen e.g. in the caller?

Reply:
If we hit this path should we also print something along the lines "some additional manual cleanup may be required"?

Reply:
>in the case where they are both printed:

You are right about the 2 messages, but I guess is case of failure I should just return and never print "deletion forced...".

>Should we also log mainErr or does that already happen e.g. in the caller?

`mainErr` is handled by cobra afterwards, as we are running a `RunE` 👍 

> If we hit this path should we also print something along the lines "some additional manual cleanup may be required"?

I don't know about that, because it may be a privilege issue, so maybe the user needs to be root and retry. I let the user figure why it fails, as we print the error too.

Reply:
> Can we e2e test this somehow? Do we have a way to provoke a normal uninstall failure?

I'll do that in a followup 👍

Reply:
> You are right about the 2 messages, but I guess is case of failure I should just return and never print "deletion forced...".

SGTM.

> I'll do that in a followup

Ack